TeamCode/src/main/java/com/hfrobots/tnt/season1920/DriverControls.java

```diff
@@ -222,6 +232,12 @@ public void periodicTask() {
         }
 
         kinematics.driveCartesian(xScaled, yScaled, rotateScaled, driveInverted, 0.0, useEncoders);
+
+        if (foundationGripButton.getRise()) {
```

Comment:
It's probably worth logging (Log.d(LOG_TAG, "...")) these, since we have operators that say "I didn't press the button that made it do that".